src/COMPASS/helpers.py

```diff
@@ -0,0 +1,73 @@
+'''
+collection of useful functions used across workflows
+'''
+
+from collections import defaultdict
+import os
+import pathlib
+
+from osgeo import gdal
+import h5py
+
+def check_file_path(file_path: str) -> None:
+    '''check if file path exists and raise error it does not'''
+    if not os.path.isfile(file_path):
+        err_str = f'{file_path} not found'
+        logging.error(err_str)
```

Comment:
I would propose to use `journal `as it would come for free from the `isce3` dependencies. I am not sure if we have any requirement on log and error formatting and how flexible `journal` is on that

Reply:
I have no preference. I nice part of using journal is that'll encourage more pyre use (for future workflow management?). Just need somebody to learn or who's already adept...